docs/app/pages/components/components-sections/select/select/select.component.html

```diff
@@ -275,6 +275,16 @@
     <tr uxd-api-property name="autoCloseDropdown" type="boolean" defaultValue="true">
         Determine if the dropdown panel should close on external click.
     </tr>
+    <tr uxd-api-property name="headingTemplate" type="TemplateRef">
```

Comment:
```suggestion
    <tr uxd-api-property name="optionsHeadingTemplate" type="TemplateRef">
```
`headingTemplate` is too generic a name for a component like this.